### PR TITLE
Add crds and fix patch

### PR DESCRIPTION
**Pull request by poussa**
- Add CRDs so they are also installed via `helm install`.
- Fix permission to patch secret when deleting it via `TCSClusterIssuer` deletion.

### CODE REVIEW

**Comment by poussa** [comment]
@avalluri

File: charts/trusted-certificate-issuer/templates/role.yaml [hunk 1]
```diff
@@ -81,14 +81,7 @@ rules:
   - list
   - update
   - watch
-- apiGroups:
```
Comment (avalluri) [inline charts/trusted-certificate-issuer/templates/role.yaml:84]: @poussa  The `secrets/finalizers` are needed for TCI to set/unset finalizers on the secrets it creates. So, this change is not required.

Comment (avalluri) [reply charts/trusted-certificate-issuer/templates/role.yaml:84]: @poussa Please ignore my above comment. The required 'patch' rule on secrets was missing which gets fixed by this change.

**Review by avalluri — APPROVED** [review]
Looks good to me.

**Comment by mythi** [comment]
update version to get a new release published?

**Comment by poussa** [comment]
@mythi version bump included.

**Review by mythi — APPROVED** [review]

**Pull request merged**